src/index.html

```diff
@@ -17,6 +17,87 @@
     />
```

Comment:
Add font import to html

```suggestion
    />  
    <link
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
      rel="stylesheet"
    />
```

Reply:
Thnak you :)